### ISSUE TITLE
Bare hydrogen core project fails at `(go)`

### ISSUE THREAD

**Issue by werenall** [magnetcoop/hydrogen.duct-template#42]
Steps to reproduce:
```
lein new duct testing-go +cljs +site +hydrogen/core
cd testing-go
lein repl
(dev)
(go)
```

I get this error:
```
user=> (dev)
:loaded
dev=> (go)
Execution error (ClassNotFoundException) at jdk.internal.loader.BuiltinClassLoader/loadClass (BuiltinClassLoader.java:583).
com.fasterxml.jackson.core.util.JacksonFeature
```

I hot-fixed it by adding this dependency to project.clj:
`[com.fasterxml.jackson.core/jackson-core "2.12.3"]`

How did I know that that's the issue?
`lein deps :tree`
![image](https://user-images.githubusercontent.com/18025307/123227743-2c393000-d4d5-11eb-93d2-b94b2fcd85f9.png)

But that shouldn't be necessary to do in *product project*. This dependency conflict should probably be fixed either in the template or the core module.

**Comment by iarenaza** [comment]
I'm afraid it's broken because base Duct template upgraded the version of `duct.module/web`, which in turn upgraded to a more recent `muuntaja` version, that in turn needs a more recent `jackson` library (because it uses something that is not available in older versions).

But our own Hydrogen duct template `cljs-ajax` dependency depends on an older version of `jackson`, and that ones happens to be the one chosen by Leiniengen (hence the error).

I have a local commit for that in `hydrogen.duct-template`. I add a direct dependency for `metosin/jsonista` v 0.3.3, which is what `duct.module/web` also expects, and is also what `muuntaja` also uses. I also add it to be our default JSON manipulation library. It's both faster and more featureful than `clojure.data.json` and `dakrone/cheshire`, so I thought it would kill two birds with one stone.

**Comment by iarenaza** [comment]
Fixed in 6737dd59
